### PR TITLE
Added links to yaml and md websites

### PR DESCRIPTION
**Pull request by rnyb**
A minor extension to webviz example config

### CODE REVIEW

File: webviz_examples/webviz-full-demo.yml [hunk 1]
```diff
@@ -403,18 +403,15 @@ layout:
...
             content:
               - SeismicMisfit:
                   ensembles: *hm_ensembles
                   attributes:
                     - topvolantis_amplitude_mean_20200701_20180101_1.txt
                     - basevolantis_amplitude_mean_20200701_20180101_1.txt
-                  # attribute_sim_path: sim2seis/output/4d_attribute_maps/  # relative to <runpath>
-                  # attribute_obs_path: ../../share/observations/seismic/  # relative to <runpath>
+                  attribute_sim_path: sim2seis/output/4d_attribute_maps/  # relative to <runpath>
+                  attribute_obs_path: ../../share/observations/seismic/  # relative to <runpath>
```
Comment (anders-kiaer) [inline webviz_examples/webviz-full-demo.yml:414]: Is this change intentional?

**Review by anders-kiaer — APPROVED** [review]

File: webviz_examples/webviz-full-demo.yml (hunk 1 above)
Comment (rnyb) [reply webviz_examples/webviz-full-demo.yml:414]: Yes, it’s a preparational step to enable a change of the default path values in the plugin code without breaking the live demo.
